backend/corpora/lambdas/api/v1/common.py

```diff
@@ -2,11 +2,15 @@
 
 from backend.corpora.common.corpora_orm import CollectionVisibility
 from backend.corpora.common.entities import Collection, Dataset
-from backend.corpora.common.utils.http_exceptions import ForbiddenHTTPException, MethodNotAllowedException
+from backend.corpora.common.utils.http_exceptions import (
+    ForbiddenHTTPException,
+    MethodNotAllowedException,
+    InvalidParametersHTTPException,
+)
 from backend.corpora.lambdas.api.v1.authorization import owner_or_allowed
 
 
-def get_collection(db_session, collection_uuid, **kwargs):
+def authorize_get_collection(db_session, collection_uuid, **kwargs):
```

Comment:
I would call this `get_collection_if_authorized` or `get_collection_authorized`

Reply:
`get_authorized_collection_or_error`

Reply:
I think if this is meant to be an auth-gated call for collections, `token_info` should be a required parameter ala `delete_dataset_common` below. And at that point, I think the required param would speak for itself and we could stick to a consistent, simpler `get_collection_common`. 

Alternatively, if we need to keep the auth parameters optional for some use case, we probably shouldn't reference auth in the name (and maybe consider making separate function for unauth and auth use cases)

Reply:
thinking about it more, I do see the value of being descriptive with a name like `get_authorized_collection` so I could be convinced to go that route. but I do think it should require `token_info` if so

Reply:
change name to get_collection_else_forbidden

Reply:
@Bento007 should we change the name of `delete_dataset_common` to `delete_dataset_else_forbidden_common`? It's not crucial, and at some point these names become cumbersome, but it is worth considering that it uses the same pattern as `get_collection_else_forbidden`. 

Generally-speaking I think we have more refactoring / tech debt to take care of wrt standards for how our methods act on our various entities, but not something we necessarily need to address before we wrap up this project, so it can wait

Reply:
also... I'm a little confused: how did this all come up in this PR? None of this relates to getting the status for a dataset, does it... 😅

Reply:
it seems like a small, uncontroversial, and beneficial change. I'm going to leave things as is for now. I don't want to introduce any additional controversial changes in this PR.